src/ui/browserAction/popup-placeholder.js

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { html, css, LitElement } from "../../vendor/lit-all.min.js";
+/**
+ * This Element shows a Placeholder Animation until the
+ * Backend Script is finalized, after finalization, it will remove itself from the DOM.
+ */
+export class PopupPlaceHolder extends LitElement {
+  static properties = {
+    // Show placeholder until this promise is resolved.
+    untilPromise: { type: Object },
+  };
+  constructor() {
+    super();
+    // This module uses top-level await, so if this module is loaded,
+    // we know ipc is working
+    this.untilPromise = import("./backend.js");
+  }
+  connectedCallback() {
+    super.connectedCallback();
+    this.untilPromise.then(() => {
+      // Goodbye!
+      this.parentElement.removeChild(this);
+    });
+  }
+
+  render() {
+    return html`
+      <vpn-titlebar title="Loading"> </vpn-titlebar>
```

Comment:
Note This is a hardcoded string. This is not a view people should see, so i chose that. 
Otherwise should we use "Mozilla VPN" ? or just empty?